spec/lib/external_count_fetcher_spec.rb

```diff
@@ -33,7 +33,7 @@ def counts_hash(count: )
         expect(count_fetcher.house_supporters.ttl).to eq(ExternalCountFetcher::REDIS_EXPIRE_SECONDS)
       end
       Timecop.freeze do
-        new_time = (ExternalCountFetcher::REDIS_EXPIRE_SECONDS-20.minutes.to_i)
+        new_time = (ExternalCountFetcher::REDIS_EXPIRE_SECONDS/3)
```

Comment:
I should have been more explicit - as now I cannot remeber this fully.  At some point there was a minor bug somewhere along the redis pipe that was causing our app to silently fail to set expeirations.  I'm going to leave it alone for now.